utils/python/create_test_impl.py

```diff
@@ -425,7 +421,7 @@ def _handle_test_status_file(self, test_name, test_phase, success):
             # TODO: What to do here? This failure is very severe because the
             # only way for test results to be communicated is by the TestStatus
             # file.
-            warning("VERY BAD! Could not handle TestStatus file '%s': '%s'" %
```

Comment:
This error should be critical. I only used warning because I couldn't use expect(False, ...) due to threading issues.